modules/ROOT/pages/install-sys-reqs.adoc

```diff
@@ -95,7 +95,9 @@ To use Runtime Fabric in a production environments, you must also have the follo
 
 * An external load balancer: required if incoming traffic is enabled to Runtime Fabric. A TCP
 load balancer with a server pool consisting of each of the controller nodes is sufficient.
-* At least one additional worker node, including hard disks. This node should be used when:
+* At least one additional worker node, including hard disks. If preserving uptime for the cluster and applications 
```

Comment:
Can this be worded from the audience point of view, or is this backend?

     "If you must preserve uptime..., you must add [have available?] at least one worker node when updating.... 
      You must also add a node when performing rolling upgrades...or OS-level patches."

Reply:
Made some updates to (hopefully) improve the wording. Thank you!